packages/blocks/src/root-block/commands/block-crud/get-selected-blocks.ts

```diff
@@ -48,7 +48,21 @@ export const getSelectedBlocksCommand: Command<
     const viewStore = ctx.std.view;
     const selectedBlockElements = blockSelections.flatMap(selection => {
       const el = viewStore.viewFromPath('block', selection.path);
-      return el ?? [];
+      if (!el) {
```

Comment:
We need to implement options like `types` and `roles` for `BlockSelection` as well. (Just like what we do for `TextSelection`)